functests/5_latency_testing/5_latency_testing_suite_test.go

```diff
@@ -29,26 +30,64 @@ import (
 	ginkgo_reporters "kubevirt.io/qe-tools/pkg/ginkgo-reporters"
 )
 
+//TODO get commonly used variables from one shared file that defines constants
+const filePath = "../../build/_output/bin/latency-e2e.test"
+
 var prePullNamespace = &corev1.Namespace{
 	ObjectMeta: metav1.ObjectMeta{
 		Name: "testing-prepull",
 	},
 }
 var profile *performancev2.PerformanceProfile
 
-var _ = BeforeSuite(func() {
+func Test5LatencyTesting(t *testing.T) {
```

Comment:
Is it a good idea to drop the Ginkgo features (Before, After etc?). Won't we lose functionality like focusing or something else this way?

Reply:
no, those are optional nodes, so ginkgo features will still work with having as minimal as the main TestXXX(t *testing.T) only.